docs/configure/babel.md

```diff
@@ -133,7 +133,9 @@ When the command finishes running, it will display the available Babel configura
 
 For more info, please refer to the [Babel documentation](https://babeljs.io/docs/en/configuration#print-effective-configs).
 
-### SWC alternative (experimental)
+### SWC alternative
+
+(🧪 Experimental)
```

Comment:
this experimental notice has another format than the one you added for the indexers above.